src/lib/outgoing_message.ts

```diff
@@ -29,6 +29,8 @@ interface OutgoingResultMessageError {
 
 export type ResultTypes = {
   start_listening: { state: ZwaveState };
+} & {
+  update_log_config: Record<string, never>;
 } & NodeResultTypes &
```

Comment:
I think that this should be an `|` ?

```suggestion
} | {
  update_log_config: Record<string, never>;
} | NodeResultTypes |
```

I also think that we want to extract the 2 anonymous defined interfaces into named interfaces.

Reply:
I followed the existing convention by using `&` but `|` makes more sense if I understand this correctly. How was this not broken before?

Reply:
I think it's just used for typing but probably not very useful haha

Reply:
looks like `&` is right, `|` results in type errors